### DISCUSSION TITLE
Is an AMM intended to be caught under the new definition of "virtual asset trading platform"?

### DISCUSSION THREAD

**Discussion by ghcproject** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#21] [Q&A]
Is an AMM intended to be caught under the new definition of "virtual asset trading platform"?

**Comment by FinalTwist** [comment]
What do you mean by AMM?  Could you describe the functions of an AMM in the context of an exchange?

**Comment by JupiterBlock** [comment]
I would say it depends?

AMMs use liquidity pools, where users can deposit assets to provide liquidity (aka they provide a liquidity service). The liquidity pool is locked in a smart contract, allowing other traders demanding liquidity (liquidity-takers) to buy or sell the assets in the pool. The execution of trades in AMMs requires validations based on the blockchain's consensus protocol, with validators adding the transactions to the blockchain.  Even if an operator could be identified for user interfaces or maintenance or upgrade operations, it's still a peer‐to‐peer model of transacting, which technically falls outside of the current definition of "virtual asset trading platform".

To be noted that similarly, DeFi protocols are interpreted to be exempt from the scope of MiCA (aka EU regulation) when they are “fully decentralized” — meaning no person or company controls the platform being used. However, ESMA, in a consultation paper for MiCA published in October 2023, noted that transparency for AMMs should include the mathematical formula used to determine the price in the liquidity pool and, if applicable, a price simulator, which implies that at least some form of guidance of AMMs will be provided in the future. Also, in 2023 IOSCO noted that AMMs may be engaging in custodial activities and acting as counterparties, depending on how the assets are transferred to the smart contracts. 

In a recent article, _On DeFi and On-Chain CeFi: How (Not) to Regulate Decentralized Finance_, by Katrin Schuler, Ann Sofie Cloots and Fabian Schär, published in the Journal of Financial Regulation, 2024, 00, 1–30, https://doi.org/10.1093/jfr/fjad014, the authors introduce the concept of ‘**on-chain CeFi’** for **centralized, smart contract-based financial services** and show how centralization may occur on all layers of the DeFi stack and how it can be inherited. Their suggestion is to regulate blockchain-based financial services in the presence of centralization, and maintain regulatory focus on the on- / off-ramps between DeFi, on-chain CeFi, and (traditional) CeFi, while allowing DeFi developers to continue to innovate and build.
